step_09/ReadMe.md

```diff
@@ -1,8 +1,10 @@
 > You can download the code of this step [here](../src/step_09.py) or all the steps [here](https://github.com/Avaiga/taipy-getting-started-core/tree/develop/src).
 
-# Step 9: Scenario subscription
+# Scenario subscription
 
-This step reuses the configuration provided in step 7 except for the scenario configuration. To have an action after changing a job status, we can [subscribe a function](https://docs.taipy.io/en/latest/manuals/core/entities/scheduling-and-job-execution/#subscribe-to-job-execution) to a scenario. A status change will call this function. This feature allows the creation of logs or particular events for Taipy GUI.
+*Time to complete: 15min; Level: Advanced*
+
+To have an action after changing a job status, we can [subscribe a function](https://docs.taipy.io/en/latest/manuals/core/entities/scheduling-and-job-execution/#subscribe-to-job-execution) to a scenario. A status change calls this function. This feature allows the creation of logs or particular events for Taipy GUI.
 
 ```python
 def callback_scenario(scenario, job):
```

Comment:
What about changing this example to add a Gui notification when a job is completed?
Also please don't use `if job.status.value == 7:` but `if job.status == Status.COMPLETED:`

Reply:
Not sure how to do that